base/linalg.jl

```diff
@@ -212,6 +212,7 @@ include("linalg/exceptions.jl")
 include("linalg/generic.jl")
 
 include("linalg/blas.jl")
+import .BLAS: gemv! # consider renaming gemv! in matmul
```

Comment:
Right, makes sense, thanks. Though there is the question of whether we should have a Julia API that includes the additional scalar-multiple inputs, and if so whether or not they should match the Fortran ordering.